ann_benchmarks/algorithms/opendistroknn.py

```diff
@@ -0,0 +1,110 @@
+import logging
+from time import sleep
+from urllib.error import URLError
+from urllib.request import Request, urlopen
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+
+from ann_benchmarks.algorithms.base import BaseANN
+
+# Configure the logger.
+logging.getLogger("elasticsearch").setLevel(logging.WARN)
+
+def es_wait():
```

Comment:
We can probably import this from the other ES module, `from .elasticsearch import es_wait` to avoid duplicating it a third time.

Reply:
done